### DISCUSSION TITLE
Ui is missing

### DISCUSSION THREAD

**Discussion by NathanCenter** [dialogic-godot/dialogic#2154] [Q&A] [answered]
Hello,
I am having an odd issue, has this happened to anyone before. I was working on this game project with a buddy, and we are using Dialogic 2 to create talking heads. The UI is missing from the menu, and I am unsure what I did wrong.

![image](https://github.com/dialogic-godot/dialogic/assets/34755641/be5bb84e-3b1f-4cc3-9752-84219e6ac0ca)

Is it installed wrong or something else?
Thank you

**Comment by CakeVR** [comment]
Hello, have a look at this Dialogic doc page: https://docs.dialogic.pro/getting-started.html

**Reply by NathanCenter** [reply]
I tried uninstalling and reinstalling by following the instructions from the docs of it, however that doesn't seem to work at all. This is very odd because was working last week. This issue popped up this week.

**Reply by coppolaemilio** [reply]
What version of Godot are you using? it is not showing up on your screenshot.

**Comment by NathanCenter** [comment]
![image](https://github.com/dialogic-godot/dialogic/assets/34755641/621836f4-45d6-4a51-9918-fe33b9d94192)
We are using 4.2.1

**Reply by NathanCenter** [reply]
looking at the console this looks very odd. Why would it say that even after I uninstalled and reinstalled it.

![image](https://github.com/dialogic-godot/dialogic/assets/34755641/d277c3e3-dae3-4f29-a12f-1234029db855)

**Comment by NathanCenter** [comment, marked as answer]
Ok so the issue was the .godot folder I deleted it and reloaded the project now it works.

![image](https://github.com/dialogic-godot/dialogic/assets/34755641/60059dd8-9d21-40e2-a366-3c7b91b6f11f)
